app/celery/letters_pdf_tasks.py

```diff
@@ -137,12 +142,23 @@ def collate_letter_pdfs_to_be_sent():
     print_run_deadline = print_run_date.replace(
         hour=17, minute=30, second=0, microsecond=0
     )
+    letters_volume = {}
+
     for postage in POSTAGE_TYPES:
         current_app.logger.info(f"starting collate-letter-pdfs-to-be-sent processing for postage class {postage}")
         letters_to_print = get_key_and_size_of_letters_to_be_sent_to_print(print_run_deadline, postage)
 
+        letters_volume[postage] = 0
+        letters_volume[postage + '_sheets'] = 0
+
         for i, letters in enumerate(group_letters(letters_to_print)):
-            filenames = [letter['Key'] for letter in letters]
+            letters_volume[postage] += len(letters)
+
+            filenames = []
+            for letter in letters:
+                filenames.append(letter['Key'])
+                letters_volume[postage + '_sheets'] += letter['Size']
```

Comment:
`letter['size']` represents file size rather than the sheet count of the letter.

Reply:
ohh, ok, good to know, I will add billable units to the query instead. I thought I was being smart hahah :D